net.sonic_pi.SonicPi.json

```diff
@@ -154,10 +165,13 @@
             "name": "sonicpi",
             "buildsystem": "simple",
             "build-commands": [
-                "cd app && ./linux-prebuild.sh --build-aubio",
-                "cd app && ./linux-config.sh --config RelWithDebInfo",
-                "cmake --build app/build"
+                "cd app && ./linux-build-all.sh --config RelWithDebInfo && ./linux-release.sh"
             ],
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
```

Comment:
The new build process downloads the dependencies (that's what makes it simpler in part, I guess) using `vcpkg`, so the network needs to be available during build.

Reply:
THIS IS NOT ALLOWED.

I have tried for weeks to get something working. But offline doesn't work.

Reply:
I see now that you can't do this in the sandbox environment (the build failed because of that https://buildbot.flathub.org/#/builders/19/builds/13456)... I'll try to get around it, but any idea is welcome!

Reply:
this #12. I hadn't pushed it because it didn't work, so I just did.